### PR TITLE
Problem: Missing example of .terminator attribute usage

### PR DESCRIPTION
**Pull request by opedroso**
Solution: added example of .terminator attribute usage

### CODE REVIEW

**Comment by hintjens** [comment]
This PR has merge conflicts; can you resolve them and then push with
--force? Thanks.

File: README.txt [hunk 1]
```diff
@@ -1183,6 +1214,9 @@ GSL helps you make neat COBOL code by automatically filling the first six charac
 
 GSL uses as its line terminator the value of of the attribute `terminator` of the gsl scope.  The default value is "\n" but it could also be set to "\r\n", for example.
 
+    \# for Windows batch scripts
+    \.terminator="\n\r"
```
Comment (jimklimov) [inline README.txt]: Shouldn't this be `\r\n` not `\n\r`?

File: README.txt [hunk 2]
```diff
@@ -60,6 +60,37 @@ Cd to `src` and run:
     CCNAME=gcc47 gmake
     gmake install
 
+#### Building on Cygwin
```
Comment (jimklimov) [inline README.txt]: This looks like a good addition, but belongs in a different commit (if not PR) as it addresses a separate improvement than the subject states.

File: README.txt (hunk 1 above)
Comment (evoskuil) [inline README.txt]: yes

File: README.txt (hunk 2 above)
Comment (evoskuil) [inline README.txt]: Please feel free to fix up and reissue, thanks!

**Comment by jimklimov** [comment]
It seems the text is already there in the repo, so I posted #148 just to address the typo.
